app/test_common.go

```diff
@@ -148,6 +151,22 @@ func (tApp TestApp) CheckBalance(t *testing.T, ctx sdk.Context, owner sdk.AccAdd
 	require.Equal(t, expectedCoins, coins)
 }
 
+// Create a new auth genesis state from some addresses and coins. The state is returned marshalled into a map.
+func (tApp TestApp) NewAuthGenState(ctx sdk.Context, cdc codec.JSONCodec, addresses []sdk.AccAddress, coins []sdk.Coins) GenesisState {
```

Comment:
are we able to use the helper `NewFundedGenStateWithSameCoins` instead? it seems all usage cases of this utils here have the same coins too.

Reply:
Yep seems like it! nice catch